src/utils/arrayFetchFromConfig/arrayFetchFromConfig.ts

```diff
@@ -8,14 +8,18 @@ const arrayFetchFromConfig = (config: Configuration, queryType: Querytype): Arra
                 key: 'rest.use_refactored_array_open',
                 value: 'true'
             },
+            {
+                key: 'rest.use_refactored_array_open_and_query_submit',
```

Comment:
I believe this is setting the default? Does this PR finish the implementation to the point where this can be enabled by default?

Reply:
This PR is tested with TileDB 2.15 for the ArrayOpen (still query v3 pending)

Reply:
This config key is to enable v3 queries, I don't think we should default it to true.

Reply:
That makes sense, i can set it to false for now and re-enable it in the following PR